Space Navigator Unity project/Assets/SpaceNavigator/Plugins/SpaceNavigatorWindows.cs

```diff
@@ -11,29 +11,39 @@ class SpaceNavigatorWindows : SpaceNavigator {
 		private const float TransSensScale = 0.0001f, RotSensScale = 0.0008f;
 
 		// Public API
-		public override Vector3 GetTranslation() {
+		public override Vector3 GetTranslation()
+		{
+		    if (SubInstance._sensor != null && Translation == null)
```

Comment:
Match the indent of the rest of this method, especially since you're not using `{` `}` (which I would highly recommend using).

Reply:
Here is a real-world example of why you should really use curly braces (TLDR: _major_ security failure in Apple's SSL).
https://blog.codecentric.de/en/2014/02/curly-braces/